src/ResponsiveProvider.tsx

```diff
@@ -17,11 +23,11 @@ ResponsiveProvider.propTypes = {
   /** The names and values of the responsive breakpoints */
   thresholdMap: PropTypes.object,
   /**  @ignore */
-  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired
+  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
```

Comment:
Shouldn't `propTypes` be replaced with interfaces?

Reply:
Technically they are not identical and still have value in non-TS apps. I was trying to change as little functionality as possible in this PR.